docs/codeql/codeql-workshops/csharp/top-down-vulnerability-guide.md

```diff
@@ -0,0 +1,132 @@
+# A Top-Down Guide to Vulnerability Searches
+
+The following uses some extracts from yesterday's [ZipSlip](https://git.io/JfeOs) problem. While yesterday's presentation worked through details, this is a high-level overview and guide.
```

Comment:
Clean up text: use a relative link to the other workshop document instead of talking about dates. Also avoid git.io.

Reply:
@adityasharad I see this url is for a secret gist, (since you are the owner of that gist) just wanted to double check before adding the url for it in here, do we want to expose that? or should this url just be rm'd?

Reply:
There should be a markdown file in the repo with equivalent contents to that gist (or we should create one if it doesn't exist). Then replace the gist URL with a relative link to the markdown file within the repo.